### PR TITLE
Multiple endpoints

### PR DESCRIPTION
**Pull request by degemer**

### CODE REVIEW

File: lib/dogapi/v1/comment.rb [hunk 1]
```diff
@@ -9,36 +9,24 @@ class CommentService < Dogapi::APIService
...
-        begin
-          if options.empty?
-            raise "Must update something."
-          end
-
-          request(Net::HTTP::Put, "/api/#{API_VERSION}/comments/#{comment_id}", true, nil, options, true)
-        rescue Exception => e
-          suppress_error_if_silent e
+        if options.empty?
+          raise "Must update something."
```
Comment (olivielpeau) [inline lib/dogapi/v1/comment.rb]: Technically changes the behavior here as this exception won't be rescued anymore

Comment (olivielpeau) [inline lib/dogapi/v1/comment.rb]: (we could remove this conditional altogether, it's not really consistent to check this in this method when all the other methods make zero checks)

File: lib/dogapi/v1/comment.rb [hunk 2]
```diff
@@ -9,51 +9,24 @@ class CommentService < Dogapi::APIService
...
-
-          if options.empty?
-            raise "Must update something."
-          end
-
-          request(Net::HTTP::Put, "/api/#{API_VERSION}/comments/#{comment_id}", params, options, true)
-        rescue Exception => e
-          suppress_error_if_silent e
+        if options.empty?
+          raise "Must update something."
```
Comment (olivielpeau) [inline lib/dogapi/v1/comment.rb]: Re-adding my comment here: this exception is not rescued anymore, I think we should remove it altogether for consistency

Comment (degemer) [inline lib/dogapi/v1/comment.rb]: Thanks, I'll remove it.

**Comment by olivielpeau** [comment]
Looks good! Lots of 💚  for all the nice tests!

Have you looked into the problem that rubocop detects (`unnecessary_double_quotes`)? I think we could ignore it

**Comment by degemer** [comment]
Thanks for the review ! I addressed your comment, and added something else: when one endpoint fails, it doesn't affect the other. Could you please have one more look ? 🙇

**Comment by olivielpeau** [comment]
I had missed that, looks better now indeed! 🚀 

Feel free to merge! (once the rubocop errors are sorted out ;) )

**Comment by degemer** [comment]
`unnecessary_double_quotes` is actually a `tailor` warning. I added `rubocop` for `spec/`, but I'm not going to enable it for `lib/` for now (because there would be too many "errors"). It'll be for later.

I'm going to ignore this `tailor` warning, and merge!

**Pull request merged**